Mathlib/NumberTheory/RamificationInertia.lean

```diff
@@ -793,4 +793,65 @@ theorem sum_ramification_inertia (K L : Type*) [Field K] [Field L] [IsDedekindDo
 
 end FactorsMap
 
+section tower
+
+variable {R S T : Type*} [CommRing R] [CommRing S] [CommRing T]
+
+theorem ramificationIdx_tower [IsDedekindDomain S] [DecidableEq (Ideal S)]
+    [IsDedekindDomain T] [DecidableEq (Ideal T)]{f : R →+* S} {g : S →+* T}
+    {p : Ideal R} {P : Ideal S} {Q : Ideal T} [hpm : P.IsPrime] [hqm : Q.IsPrime]
+    (hf0 : map f p ≠ ⊥) (hg0 : map g P ≠ ⊥) (hfg : map (g.comp f) p ≠ ⊥)
+    (hp0 : P ≠ ⊥) (hq0 : Q ≠ ⊥) (hg : P = comap g Q) :
+    ramificationIdx (g.comp f) p Q = ramificationIdx f p P * ramificationIdx g P Q := by
```

Comment:
Is it possible to minimize some of these hypotheses? For instance, if `Q = ⊥`, then aren't both sides equal to zero?